app/controllers/staff/training_sessions_controller.rb

```diff
@@ -0,0 +1,130 @@
+class Staff::TrainingSessionsController < StaffAreaController
+  before_action :load_user, only: [:show, :edit, :update]
+
+  layout 'staff_area'
+
+  def index
```

Comment:
I'd actually prefer if you removed it. Confusing code we don't need right now is worse than no code